src/arch/svm/SpokeUtils.ts

```diff
@@ -613,9 +619,20 @@ export async function getFillRelayTx(
     depositId: new Uint8Array(intToU8Array32(relayData.depositId.toNumber())),
     fillDeadline: relayData.fillDeadline,
     exclusivityDeadline: relayData.exclusivityDeadline,
-    message: new Uint8Array(Buffer.from(relayData.message, "hex")),
```

Comment:
I don't see how this wasn't a bug before, which is bad, since we could fill relays before this.